client/src/components/ResultPanel/index.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { Disposable, Uri, commands, window } from "vscode";
+
+import { createWriteStream } from "fs";
+
+import { SubscriptionProvider } from "../SubscriptionProvider";
+import { ResultsContext, resultsHtml } from "./ResultPanel";
+
+export class ResultPanelSubscriptionProvider implements SubscriptionProvider {
+  getSubscriptions(): Disposable[] {
+    return [
+      commands.registerCommand(
+        "SAS.saveHTML",
+        async (context: ResultsContext) => {
+          const html = resultsHtml[context.uuid] || "";
+          if (!html) {
+            return;
+          }
+          const uri = await window.showSaveDialog({
+            defaultUri: Uri.file(`results.html`),
```

Comment:
nit: I guess it would be great if we can prepend the SAS code file name to it as SAS Studio does.

Reply:
It seems like that could be more of a challenge since we could have saved files, and files that have yet to be saved. That said, we could, at the very least, make it such that the default uri better matches the webview panel title. I'll look into that.